pkg/apis/v1/ec2nodeclass.go

```diff
@@ -324,11 +324,11 @@ type MetadataOptions struct {
 
 type BlockDeviceMapping struct {
 	// The device name (for example, /dev/sdh or xvdh).
-	// +required
+	// +optional
```

Comment:
why is this changing?

Reply:
controller-gen was not generating the required field on v0.15.0. This now will result in a breaking change for customers if we require block device mapping fields https://github.com/kubernetes-sigs/controller-tools/pull/944